apps/01PigLatin.js

```diff
@@ -7,11 +7,83 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+//Concept check
+//
+//Please explain why we are using String.prototype.toLowerCase() here.
+//For more info:
+//https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+//
+//Your Answer:
+//It is used to convert a string to lowercase letters, but it does not change the orginal string.
 
-}
+word = word.toLowerCase();
+
+//Concept Check
+//
+//What is the significance of setting vowelIndex to -1?
+//This might help:
+//https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+//
+//Your Answer:
+//To move any consonant from the beggining of the word to the end of it (e.g. cat = atc).
+
+var vowelIndex = -1;
+
+if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+             vowelIndex = word.indexOf('a');
+         }
+
+if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+             vowelIndex = word.indexOf('e');
+         }
+
+if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+             vowelIndex = word.indexOf('i');
```

Comment:
Thank you! Will make the necessary corrections. 

Sent from my iPhone

> On Aug 28, 2016, at 9:45 AM, Codacy Bot <notifications@github.com> wrote:
> 
> In apps/01PigLatin.js:
> 
> > +//
> > +//Your Answer:
> > +//To move any consonant from the beggining of the word to the end of it (e.g. cat = atc).
> > +
> > +var vowelIndex = -1;
> > +
> > +if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
> > +             vowelIndex = word.indexOf('a');
> > +         }
> > +
> > +if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
> > +             vowelIndex = word.indexOf('e');
> > +         }
> > +
> > +if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
> > +             vowelIndex = word.indexOf('i');
>  Issue found: Strings must use doublequote. (quotes)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>